src/test/java/gmbs/model/CarTest.java

```diff
@@ -0,0 +1,61 @@
+package gmbs.model;
+
+import static org.assertj.core.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+class CarTest {
+
+    private static final String DUPLICATED_NAME_MESSAGE = "[ERROR] 중복된 이름이 있습니다";
+
+    @Test
+    void 자동차_생성() {
+        // given
+        String name = "test1";
+
+        // when
+        // then
+        assertThatCode(() -> {
+            new Cars(name);
+        }).doesNotThrowAnyException();
```

Comment:
```suggestion
        assertThatCode(() -> new Cars(names))
                .doesNotThrowAnyException();
```
{} 괄호가 없어도 되는데 왜 추가하셨을까요~?

Reply:
수정했습니다